src/components/app/Home.js

```diff
@@ -542,6 +579,10 @@ function _isFirefox(): boolean {
   return Boolean(navigator.userAgent.match(/Firefox\/\d+\.\d+/));
 }
 
+function _isAndroid(): boolean {
+  return Boolean.apply(navigator.userAgent.match(/Android\b/));
```

Comment:
If the only thing you want to know is whether the regexp match, it's better to use the funciton [regexp.test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test). In general it's better to use regexps methods, because if the string is undefined or null this won't throw.

In this case, you can do this:
```js
return /Android\b/.test(navigator.userAgent);
```
And you don't need to cast to Boolean because it's already boolean.

This is also likely faster because the regexp engine doesn't try to capture more information (in this case this isn't visible of course, because the regexp is simple and the string is small).

One quick question: why have you used the anchor `\b` here? I'm not opposed, just wanted to know the rationale.